studybuddies/src/app/components/error-page/error-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ErrorPageComponent } from './error-page.component';
+
+describe('ErrorPageComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
describe("ErrorPageComponent", () => {
```